### DISCUSSION TITLE
Task Priority

### DISCUSSION THREAD

**Discussion by GillesLACAUD** [marcel-licence/esp32_multitrack_looper#3] [Q&A]
Hi Marcel
Are you sure for the task priority
You set 999....and i think it is a higher priority....
In my application for a status task i set the priority to 0

[https://techtutorialsx.com/2017/05/09/esp32-running-code-on-a-specific-core/](url)

**Comment by marcel-licence** [comment]
Hi,
hm I just didn't care because it seems the task is alone on this core. So it should run alone.
It didn't show any issues. It was just for the displayed messages (terminal output). 
I found information that a higher number represents a higher priority so it was intentional =)
Best, Marcel

**Reply by GillesLACAUD** [reply]
for me when i generate a sine wave i can hear the difference...some click in the signal.
But yes your idea is right....strange.

**Reply by marcel-licence** [reply]
Which hardware do you use? Do you use the blocking write method like in my code?
I keept intentionally the audio task complete blocking to avoid any problems like this. But it is far from optimum at the moment.
